modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -65,23 +66,24 @@ To access the KV (Key/Value) API or to query views, you need to open a `Bucket`:
 include::example$start_using.rb[tag=bucket,indent=0]
 ----
 
-// If you installed the travel sample data bucket, substitute _travel-sample_ for _bucket-name_.
+If you installed the travel sample data bucket, substitute _travel-sample_ for _bucket_name_.
```

Comment:
did you verify that the markup (e.g. italics vs underscore) work correctly here, I can't remember how antora handles this, but has bitten me with other systems in the past...

Reply:
Yep works fine 👍